docs/api/classifier/confusion_matrix.rst

```diff
@@ -66,7 +66,7 @@ Class names can be added to a `ConfusionMatrix` plot using the `label_encoder` a
     y = iris.target
     classes = iris.target_names
 
-    X_train, X_test, y_train, y_test = tts(X, y, test_size=0.2)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)
```

Comment:
Changed `tts` to `train_test_split` since the function is imported above using `from sklearn.model_selection import train_test_split`